tests/version_test.py

```diff
@@ -25,10 +28,29 @@ def get_version():
     return pyproject["tool"]["poetry"]["version"]
 
 
+def get_conda_recipe_version():
```

Comment:
can you please also update the repo templates so that they have that test? (new JIRA issue?)
Possibly mark the test as skipped it the meta.yml file does no exist?